lib/routes/home/widgets/payments_list/widgets/payment_item/widgets/payment_item_amount.dart

```diff
@@ -23,19 +23,31 @@ class PaymentItemAmount extends StatelessWidget {
           final bool hideBalance = userModel.profileSettings.hideBalance;
           return BlocBuilder<CurrencyCubit, CurrencyState>(
             builder: (BuildContext context, CurrencyState currencyState) {
-              final String amount = currencyState.bitcoinCurrency.format(
+              String amount = currencyState.bitcoinCurrency.format(
                 paymentData.amountSat,
                 includeDisplayName: false,
               );
 
-              final int actualFeeSat = paymentData.actualFeeSat;
+              int actualFeeSat = paymentData.actualFeeSat;
               final String actualFeeFormatted = currencyState.bitcoinCurrency.format(
                 actualFeeSat,
                 includeDisplayName: false,
               );
 
+              if (paymentData.status == PaymentState.refundPending) {
+                amount = currencyState.bitcoinCurrency.format(
+                  paymentData.amountSat + actualFeeSat,
+                  includeDisplayName: false,
+                );
+
+                actualFeeSat = 0;
+              }
```

Comment:
Is there a reason this is changed only during the pending refund? It will just go back to the original values when the refund is complete

Reply:
Addressed with:
- [Display total refundable amount on Amount widget](https://github.com/breez/misty-breez/pull/410/commits/a2a745f7a1e7ebc74a857200baaa8c32aafa2eb3)

where we display the estimated total refund amount (payment amount + fees) for:

1. pending refunds (PaymentState.refundPending)
2. completed refunds where we can't track the exact transaction amount (isRefunded && refundTxAmountSat == 0)

Reply:
Ok, I think this will only show the correct amount for amountless receive chain swaps. If an amount was selected to be received it show the expected payable amount (not actual send amount)

Reply:
Refund swaps have their amount fixed, we don't allow the user to change it on Misty Breez.

Note: Amountless swaps are not integrated yet on Misty Breez.

Reply:
That wasn't my experience, perhaps we update the payment data?

```
{
  "event": "update",
  "channel": "swap.update",
  "args": [
    {
      "id": "gh2XWg6yMouD",
      "status": "transaction.lockupFailed",
      "failureReason": "locked 26000 is more than expected 25000",
      "failureDetails": {
        "expected": 25000,
        "actual": 26000
      }
    }
  ],
  "timestamp": "1741297502609"
}
```

<img width="414" alt="image" src="https://github.com/user-attachments/assets/55dd5715-0fef-42f7-8c09-d7de2050d7e7" />

Reply:
The behavior on the example I gave was due to a bug, which got addressed in 
- https://github.com/breez/breez-sdk-liquid/pull/771

Although the  refundable amount estimation is incorrect, we are keeping these changes as it's still an improvement over previous implementation and if i'm not mistaken, it will reflect the expected amount for the failed transaction.